app/scripts/controllers/main.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+angular.module('todoListApp')
+  .controller('MainCtrl', function ($scope, localStorageService) {
```

Comment:
This form of implicit dependency injection will fail if your js file is minified.  You should use the preferred method:
```
.controller('MainCtrl', ['$scope', 'localStorageService', function ($scope, localStorageService) {
  ...
}]);
```